Client/Frontend/Browser/URIFixup.swift

```diff
@@ -50,4 +62,18 @@ class URIFixup {
         }
         return components?.url
     }
+    
+    /// Checks whether a string is a valid email conforming to RFC 2822.
+    /// http://www.cocoawithlove.com/2009/06/verifying-that-string-is-email-address.html
+    static func isValidEmail(_ string: String) -> Bool {
```

Comment:
Could we add few unit tests to it? That regex looks scary :D

Reply:
Fixed. Added Punycode unit tests.

Reply:
Added Unit tests for emails. Although, that regex conforms to RFC2822 strictly, I added on that allows more "relaxed" testing which omits quotes and square brackets which still conforms to RFC2822.